ldms/src/ldmsd/ldmsd_request.c

```diff
@@ -5933,7 +5933,9 @@ static int stream_publish_handler(ldmsd_req_ctxt_t reqc)
 	}
 
 	reqc->errcode = 0;
-	ldmsd_send_req_response(reqc, "ACK");
+	attr = ldmsd_req_attr_get_by_id(reqc->req_buf, LDMSD_ATTR_TYPE);
```

Comment:
I'm guessing that the presence of this attribute means that this is the republish path.

Reply:
Yes, only the republish path adds an LDMSD_ATTR_TYPE attribute in the stream_publish request.